public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -60,7 +60,7 @@ define(function(require) {
     if(publishUrl) {
       this.updateDialog(publishUrl, true);
     }
-
+/** TODO - not sure about this...
     if(publisher.fsync) {
```

Comment:
I think this will become:
```js
// earlier as part of the publisher object
bramble.on("fileChange", function() {
  if(publisher.needsUpdate) return;
  if(!Project.getPublishUrl()) return; // We also need to get/set this everytime we publish/unpublish below
  publisher.handlers.unpublishedChangesPrompt();
  Project.publishNeedsUpdate(true, function(err) {
    if(err) {
      console.error("[Thimble] Failed to set the publishNeedsUpdate flag after a file change: ", err);
      return;
    }
    publisher.needsUpdate = true;
  });
});
```